documents/2-RTB/Documentazione_Esterna/Analisi_dei_Requisiti.typ

```diff
@@ -195,43 +202,312 @@ L'attore coinvolto nei casi d'uso è lo #glossary("User") che accede al servizio
-  - L'utente apre l'interfaccia di #glossary("BuddyBot") e pone una domanda/richiesta.
-  - esempio: "Mostrami l'ultima guida aggiornata per il deploy."
-  - #glossary("LLM") interpreta la domanda dell'utente e invia una richiesta alle #glossary("API") di #glossary("Confluence") per ricercare le informazioni sulla base dei dettagli forniti dall'utente.
+    *Attori coinvolti*: #glossary("GUI"), #glossary("Confluence") #glossary("API").
+
+    *Inclusioni* //non sapevo se metterle come inclusioni o specializzazioni quindi ho seguito il diagramma
+    - UC3.1 Richiesta Id di una pagina
+    - UC3.2 Richiesta titolo di una pagina
+    - UC3.3 Richiesta status di una pagina
+    - UC3.4 Richiesta autore di una pagina
+    - UC3.5 Richiesta owner di una pagina
+    - UC3.6 Richiesta spazio di una pagina
+    - UC3.7 Richiesta Body di una pagina
+    - UC3.8 Richiesta Id di una Task
+    - UC3.9 Richiesta Status di una Task
+    - UC3.10 Richiesta creatore di una Task
+    - UC3.11 Richiesta assegnatario di una Task
+    - UC3.12 Richiesta data di scadenza di una Task
+
+    // *Specializzazioni*
+    // - UC3.1, Consultazione Documenti
+    // - UC3.2, Consultazione Pagina
+    // - UC3.3, Consultazione Task
+
+    *Precondizioni*:
+    - Le #glossary("API") di #glossary("Confluence") sono disponibili e configurate correttamente.
+    - La #glossary("GUI") di #glossary("BuddyBot") è attiva e pronta per l'interazione con l'utente. //BOH
+    - L'utente ha posto una richiesta non fuori contesto e conforme con lo scope del sistema
+
+    *Postcondizioni*:
+    - L'utente riceve la parte interessata di tale documento.
+
+    *Scenario principale*:
+    - Il sistema interpreta la domanda dell'utente e invia una richiesta alle #glossary("API") di #glossary("Confluence") per ricercare le informazioni sulla base dei dettagli forniti dall'utente. //anche qui solo questa?
+    - il sistema ritorna la risposta all'utente in modo chiaro e comprensibile. //BOHH
+
+    *User Story Associata*:
+    - "Come utente, voglio poter accedere alle informazioni aziendali presenti in Confluence in modo rapido e intuitivo, per poter risolvere i miei dubbi e problemi in modo efficace."
   ]
   #colbreak()
   #figure(
     image(ar.diagUC3, width: 25em, fit: "contain"),
-    caption: "Diagramma UC3, consultazione Confluence"
+    caption: "Diagramma UC3, consultazione Confluence",
   )
 ]
 
-- Dopo aver ottenuto le informazioni necessarie, #glossary("LLM") le elabora per fornire una risposta chiara, leggibile e comprensibile all'utente, facilitando a quest'ultimo l'accesso alle informazione.
+#glossary("Confluence") permette di reperire diversi tipi di informazione.
 
-*Inclusioni*
-- Reperimento delle informazioni (#glossary("API"))
-- Elaborazione dei dati ricevuti (#glossary("LLM"))
+Quelle interessate per questo progetto sono:
 
+- *Pagina*: per accedere a informazioni specifiche presenti in una pagina;
+- *Task*: per accedere a task e attività da svolgere.
```

Comment:
riga [254,259]: cambierei in-> 
"#glossary("Confluence") consente di accedere a diversi tipi di informazione. Per il progetto in questione, le tipologie di dati rilevanti sono:

Pagina: informazioni specifiche contenute all'interno di una singola pagina;
Task: dettagli relativi a task e attività da svolgere."

Reply:
done